cpp/mg_procedure/mg_procedure.h

```diff
@@ -167,6 +167,8 @@ enum mgp_value_type {
   MGP_VALUE_TYPE_DURATION,
 };
 
+enum mgp_error mgp_value_copy(struct mgp_value *val, struct mgp_memory *memory, struct mgp_value **result);
+
```

Comment:
~Why these copy functions needs to be part of the C API? Cannot they be done on the "user side"? Currently we only have copy functions for objects that are otherwise couldn't be copied. With these functions we add 3 copy functions, but not to others. My concern here is we make our API inconsistent with these, but I don't see a proper reason why are they important.~

`mgp_path` already has a copy function, so these copy functions are actually a good fit.